kafka/src/main/java/com/expedia/adaptivealerting/kafka/visualizer/Utility.java

```diff
@@ -0,0 +1,30 @@
+package com.expedia.adaptivealerting.kafka.visualizer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Utility {
```

Comment:
Please choose a classname that clarifies what sort of utility this is. Also please add Lombok `@UtilityClass`